news/_posts/core-weekly/2022-08-22-coreweekly-33-2022.md

```diff
@@ -0,0 +1,135 @@
+---
+layout: post
+title:  "PrestaShop Core Weekly - Week 33 of 2022"
+subtitle: "An inside look at the PrestaShop codebase"
+date:   2022-08-22
+authors: [ PrestaShop ]
+image: /assets/images/2017/04/core_weekly_banner.jpg
+twitter_image: /assets/images/theme/banner-core-weekly.jpg
+icon: icon-calendar
+tags:
+ - core-weekly
+---
+
+This edition of the Core Weekly report highlights changes in PrestaShop's core codebase from Monday 15th to Sunday 21th of August 2022.
+
+![Core Weekly banner](/assets/images/2018/12/banner-core-weekly.jpg)
+
+## General messages
+
+Dear developers,
+
+PrestaShop 8 Beta 1 phase continues. Remember that you have time until September 5th, 2022, to test and [report](https://github.com/PrestaShop/PrestaShop/issues/new/choose) any issues you find.
+
+Last week there was a significant update of the developer documentation that summarizes [notable changes in PrestaShop 8](https://devdocs.prestashop.com/8/modules/core-updates/8.0/#notable-changes). We highly recommend to check them.
+
+## Releases
+
+* [ps_customersignin](https://github.com/PrestaShop/ps_customersignin): [v2.0.5](https://github.com/PrestaShop/ps_customersignin/releases/tag/v2.0.5)
```

Comment:
Confirmed 👍 

```suggestion
* [Ps_customersignin](https://github.com/PrestaShop/ps_customersignin) module: [v2.0.5](https://github.com/PrestaShop/ps_customersignin/releases/tag/v2.0.5)
```

@kpodemski Before, I used to add "module" for module releases and also make first letter a capital. You decide if you want to continue 😉

Reply:
@kpodemski ok, I'll update your script that generates this list :)